app/tree/gui/src/ts/twxplore/gui/geo/components/TreeCollapse/TreeCollapse.tsx

```diff
@@ -0,0 +1,112 @@
+import './TreeCollapse.scss';
+import * as React from "react";
+import { Theme, createStyles, makeStyles } from '@material-ui/core/styles';
+import {ListGroup, ListGroupItem} from "reactstrap";
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+//import * as PropTypes from "prop-types";
+
+
+
+type Feature_List = {
```

Comment:
I'd prefer interface here.

Reply:
This is a lot of properties. Is this the same as a tree definition? Can you get this from the GraphQL API definition?

Reply:
Imported TreesQuery_trees from TreesQuery.ts and used that instead.